.github/CODEOWNERS

```diff
@@ -3,7 +3,7 @@
 * @o3de/sig-docs-community-maintainers
```

Comment:
Let's skip this file. I've got a PR that makes changes to codeowners in dev, and that will be used to update main.